src/main/java/uk/gov/companieshouse/api/accounts/util/ixbrl/accountsbuilder/SmallFullAccountsBuilderImpl.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.companieshouse.api.accounts.util.ixbrl.accountsbuilder;
+
+import java.io.IOException;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.Account;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.balancesheet.BalanceSheet;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.balancesheet.CalledUpSharedCapitalNotPaid;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.company.Company;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.notes.Notes;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.notes.PostBalanceSheetEvents;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.period.Period;
+
+@Component
+public class SmallFullAccountsBuilderImpl implements AccountsBuilder {
+
+    @Override
+    public Object buildAccount() throws IOException {
+        return getSmallFullAccount();
+    }
+
+    /**
+     * Builds the small full accounts model. Functionality needs to be change.
+     */
+    private Account getSmallFullAccount() {
+        //TODO: REMOVE HARDCODED VALUES. Use functionality to retrieve the accounts information when built in the API.
+        Account account = new Account();
+        account.setPeriod(getAccountPeriod());
+        account.setBalanceSheet(getBalanceSheet());
+        account.setNotes(getNotes());
+        account.setCompany(getCompany());
+
+        return account;
+    }
+
+    private Company getCompany() {
+        Company company = new Company();
+        company.setCompanyName("SC344891");
+        company.setCompanyNumber("MYRETON RENEWABLE ENERGY LIMITED");
```

Comment:
As part of GDPR we cannot be hardcoding real company data. This should be removed until the company name and number are available.